src/pspm_cfg/pspm_cfg_glm_hp_rew.m

```diff
@@ -0,0 +1,50 @@
+function [glm_hp_rew] = pspm_cfg_glm_hp_rew
+% GLM HP FC
+
+% Initialise
+global settings
+if isempty(settings), pspm_init; end;
+
+% set variables
+
+vars = struct();
+vars.modality = 'HPR';
+vars.modspec = 'hp_fc';
+vars.glmref = { ...
+        ['Xia Y, Liu H, Kälin OK, Gerster S, Bach DR (under review). Measuring', ...
+         'human Pavlovian appetitive conditioning and memory retention.']};
+vars.glmhelp = '';
+
+% load default settings
+glm_hp_rew = pspm_cfg_glm(vars);
```

Comment:
There is a "Nuisance File" option, and it does not have to be specified, because on the current design. I am unsure if this is intentional or not. If it is an optional file, I would suggest to add "optional" in the help text. If it is mandatory, I suggest to report an error if it is not suggested. Currently it is not mandatory and can be left black for running the module.


<img width="895" alt="image" src="https://github.com/bachlab/PsPM/assets/3895146/2239a4f1-c07b-4040-ac57-82e87fa1aca5">

Reply:
a generic stylistic issue for GLM, will be dealt with in future PR.